pkg/cmd/generate-postgres-metadata-tables/main.go

```diff
@@ -56,6 +75,9 @@ func main() {
 		if err := rows.Scan(&table, &column, &dataType, &dataTypeOid); err != nil {
 			panic(err)
 		}
+		if err := getMappedType(&dataType, &dataTypeOid); err != nil {
```

Comment:
maybe `dataType` should be renamed to `dataTypeName`

Reply:
Done.